tinygrad/runtime/lib.py

```diff
@@ -41,6 +43,14 @@ def _buffer(self) -> memoryview: raise NotImplementedError("must be implemented"
   def toCPU(self) -> np.ndarray: return np.frombuffer(self._buffer(), dtype=np.dtype(self.dtype.np, metadata={"backing": self}), count=self.size)  # type: ignore
   def _copyin(self, x:np.ndarray) -> None: np.copyto(self.toCPU(), x.reshape(-1))
 
+  @classmethod
+  def fromBuffer(cls, src, shape, dtype, **kwargs):
+    from tinygrad.runtime.ops_disk import RawDiskBuffer
+    if isinstance(src.realized, RawDiskBuffer):
+      assert all_int(shape), "does not support symbolic shape"
+      src.realized.readinto(cast(RawBufferMapped, cls(prod(shape), dtype, **kwargs))._buffer())
```

Comment:
Like this shouldn't be here, it should be in RawDiskBuffer

Reply:
@geohot, could you please elaborate? Are you suggesting adding a 'transfer' method to RawDiskBuffer (as currently done), reimplementing RawDiskBuffer as a subclass of RawBufferTransfer, or something different?